tests/test_dbus.py

```diff
@@ -695,8 +695,9 @@ class DBusForkedTestCase(DBusTestCase):
     """Test DBus support with a real DBus connection."""
 
     def setUp(self):
-        super().setUp(server_args={"server": GLibServerUnix},
-                      proxy_args={"client": GLibClientUnix})
+        super().setUp()
+        self.server_args = {"server": GLibServerUnix}
+        self.client_args = {"client": GLibClientUnix}
```

Comment:
Is it on purpose that here the variable is named `client_args` (renamed from `proxy_args`) but parent have it still as `proxy_args`? If yes, could you please explain this split in the commit message?

Reply:
It was a bug. Good catch!